datasets/UKNI01_northern_ireland/UKNI01_northern_ireland.derived.ad

```diff
@@ -18,26 +18,25 @@
 - has_mountains = true
 - has_old_technologies = true
 - has_other = true
-- has_solar_csp = false
-- has_offshore_solar = false
+- has_solar_csp = true
```

Comment:
Why is `has_solar_csp` true for this dataset? NL doesn't have CSP either and as far as I know the weather in the UK/NI is not that much better than in NL.

Reply:
Sharp. The UKNI is based on the `eu` dataset and there solar csp is true. I think not a big issue if this technology is available, it is actually available for all EU countries now